api/clients/codecs/blob_codec.go

```diff
@@ -30,6 +30,7 @@ func GenericDecodeBlob(data []byte) ([]byte, error) {
 	if len(data) <= 32 {
 		return nil, fmt.Errorf("data is not of length greater than 32 bytes: %d", len(data))
 	}
+	// TODO: why [1]? What's in [0]?
```

Comment:
@bxue-l2 you prob know this

Reply:
Thanks, added explanation comment: [bb1eb0e](https://github.com/Layr-Labs/eigenda/pull/780/commits/bb1eb0e4c7d0d9df17ae8fd8eca2900e9f190ef5)